spec/lib/myfinance/resources/receivable_account_spec.rb

```diff
@@ -110,4 +110,19 @@
       end
     end
   end
+
+  describe "#undo_receivement", vcr: true do
+    subject { client.receivable_accounts.undo_receivement(1215634, entity_id) }
+
+    it "undoes receivement of the receivable account" do
+      expect(subject.id).to eq(1215634)
+      expect(subject.status).to eq(1)
+      expect(subject.status_name).to eq('unreceived')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.